### ISSUE TITLE
AEM Author Loading Blank

### ISSUE THREAD

**Issue by YegorKozlov** [Adobe-Consulting-Services/acs-aem-commons#3463]
## Required Information

* [x] AEM Version, including Service Packs, Cumulative Fix Packs, etc: 6.5.21, 2024.9.17689.20240905T073330Z-240900
* [x] ACS AEM Commons Version: 6.9.0
* [x] Reproducible on Latest? yes/no

AEM Author is loading Blank with the latest ACS 6.9.0 

<img width="941" alt="image" src="https://github.com/user-attachments/assets/c78cb3cc-c529-4c8c-829f-cc9fd2aaf625">

@niekraaijmakers IncludeDecoratorFilterImpl seems to be the culprit. The filter is missing the `else` block

```
        if(typeCheckFn.test(RESOURCE_TYPE)){
           .... 
        }else if(resourceTypesIgnoreChildren.stream().anyMatch(typeCheckFn)){
           .... 
        } else {
           // the 'else' clause  is missing in 6.9.0
            chain.doFilter(servletRequest, servletResponse);
        }

```

**Comment by niekraaijmakers** [comment]
Thanks for spotting this @YegorKozlov . I don't know how I missed this. PR opened, https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/3464 .

**Comment by niekraaijmakers** [comment]
@davidjgonzalez  can we merge this and re-release? sorry for the bother

**Comment by jdavenport-vervio** [comment]
This issue is also caused by a syntax error in one of the errorpagehandler JSP files, I've raised a PR to fix: https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/3466

**Comment by YegorKozlov** [comment]
fixed in 6.9.4
